modules/machine-health-checks-resource.adoc

```diff
@@ -17,27 +17,30 @@ kind: MachineHealthCheck
 metadata:
   name: example <1>
   namespace: openshift-machine-api
+  annotations:
+     <2>
```

Comment:
```suggestion

```

Reply:
@kalexand-rh is this suggestion still relevant per our last discussion?

If I'll remove this, we will have `<2>` below explaining about the annotation without any reference in the yaml. 
let me know if you want to remove it anyway

Reply:
@n1r1, I think it might be clearest to provide two yaml sections.  Will you PTAL at the commit that I've added?

Reply:
@kalexand-rh thanks.
Looks good to me